tests/test_auto_testing.py

```diff
@@ -55,8 +57,29 @@ class TestableModel(Model[ModelItemType, ModelItemType]):
         def _predict(self, item):
             return item
 
-    assert list(TestableModel._iterate_test_cases()) == [
-        ("some_model", {"x": 1}, {"x": 1}, {})
+    assert list(TestableModel._iterate_test_cases()) == expected
+    assert list(TestableModel._iterate_test_cases("some_model")) == expected
+    assert list(TestableModel._iterate_test_cases("unknown_model")) == []
+
+    class TestableModel(Model[ModelItemType, ModelItemType]):
+        CONFIGURATIONS = {
+            "some_model": {
+                "test_cases": {"cases": [{"item": {"x": 1}, "result": {"x": 1}}]}
+            },
+            "some_other_model": {},
+        }
+        TEST_CASES = {"cases": [{"item": {"x": 1}, "result": {"x": 1}}]}
```

Comment:
We should get rid of the `TEST_CASES = {"cases": [...]}` structure in favor of a more straightforward `TEST_CASES=[...]`